### PR TITLE
Updated Progress table

### PR DESCRIPTION
**Pull request by Glide7**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,22 @@ We truly ❤️ pull requests! If you wish to help, you can use the templates in
 
 For security issues, kindly email us [security@appwrite.io](mailto:security@appwrite.io) instead of posting a public issue in GitHub.
 
+
+## Progress
+
+✅ = Done - You can still submit PRs with fixes or improvements. <br>
+⚒  = In progress - This function is currently being worked on by a community member.  <br>
+❌ = Missing - This funcion is missing and we are looking for contributors.
```
Comment (eldadfux) [inline README.md:38]: Missing - This funcion is missing and we are looking for contributors. => Missing - This function is missing. Contributions are welcomed.

File: README.md [hunk 2]
```diff
@@ -30,6 +30,22 @@ We truly ❤️ pull requests! If you wish to help, you can use the templates in
 
 For security issues, kindly email us [security@appwrite.io](mailto:security@appwrite.io) instead of posting a public issue in GitHub.
 
+
+## Progress
+
+✅ = Done - You can still submit PRs with fixes or improvements. <br>
+⚒  = In progress - This function is currently being worked on by a community member.  <br>
```
Comment (eldadfux) [inline README.md:37]: This function is currently being worked on by a community member. => This function is currently being worked on.

File: README.md [hunk 3]
```diff
@@ -30,6 +30,22 @@ We truly ❤️ pull requests! If you wish to help, you can use the templates in
 
 For security issues, kindly email us [security@appwrite.io](mailto:security@appwrite.io) instead of posting a public issue in GitHub.
 
+
+## Progress
+
```
Comment (eldadfux) [inline README.md:35]: Remove all the <br>

Comment (eldadfux) [reply README.md]: `<br>`

Comment (Glide7) [reply README.md]: Can you elaborate?

Comment (eldadfux) [reply README.md]: image

**Comment by eldadfux** [comment]
@Glide7, Torsten just added another example you can update

**Comment by eldadfux** [comment]
Storage cleaner is in PHP not Python

**Comment by eldadfux** [comment]
and its ready and merged.

**Comment by Glide7** [comment]
> and its ready and merged.

Fixed the table accordingly!

**Pull request merged**